vmdb/app/models/mixins/assignment_mixin.rb

```diff
@@ -85,11 +85,11 @@ def namespace
   module ClassMethods
     def assignments(find_options = {})
       # Get all assigned, enabled instances for type klass
-      ass_hash = self.all(find_options).inject({}) {|h,a| h[a.id] = a; h}
-      tags = Tag.all(:conditions => ["taggings.taggable_type = ? and tags.name like ?", self.name, "#{self.namespace}/%"], :include => "taggings")
+      assignment_map = self.all(find_options).inject({}) { |h, a| h[a.id] = a; h }
```

Comment:
No, we need to create a helper library to pull out keys from the options hash to apply: .order, .includes, .select, etc.  I think @tenderlove did it in one place in this PR.   For now, I avoided changing all/first/find with an options hash until we have that library to extract and apply the sql correctly.